apps/src/gamelab/AnimationTab/PiskelEditor.jsx

```diff
@@ -166,7 +166,7 @@ export default connect(state => ({
   selectedAnimation: state.animationTab.selectedAnimation,
   animationList: state.animationList,
   channelId: state.pageConstants.channelId,
-  allAnimationsSingleFrame: state.pageConstants.allAnimationsSingleFrame
+  allAnimationsSingleFrame: state.pageConstants.allAnimationsSingleFrame || false
```

Comment:
It's equivalent (and I think more common in our repo) to force a value to boolean like this: `!!state.pageConstants.allAnimationsSingleFrame`